spring-xd-test-fixtures/src/main/java/org/springframework/xd/test/fixtures/AvailableSocketPorts.java

```diff
@@ -53,14 +53,14 @@ public static synchronized int nextAvailablePort() {
 	/**
 	 * Verifies that the port to the broker is available. If not throws an IllegalStateException.
```

Comment:
I don't know if I'm really the original author of this, but this javadoc is really misleading as it's doing quite the opposite: it ensures that the port is *listening*